code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -213,17 +214,16 @@ descPairs IncludeUnits =
   map (\x -> (P $ eqSymb x, Flat $ phrase x +:+ sParen (toSentenceUnitless x), Nothing))
   -- FIXME: Need a Units map for looking up units from variables
 
--- | Defines 'Field's as 'String's.
-instance Show Field where
-  show Label             = "Label"
-  show Symbol            = "Symbol"
-  show Units             = "Units"
-  show RefBy             = "RefBy"
-  show Source            = "Source"
-  show Input             = "Input"
-  show Output            = "Output"
-  show InConstraints     = "Input Constraints"
-  show OutConstraints    = "Output Constraints"
-  show DefiningEquation  = "Equation"
-  show (Description _ _) = "Description"
-  show Notes             = "Notes"
+getSent :: Field -> Sentence
```

Comment:
I think the source of the problem is here: `getSent` goes immediately into "too big" a data-structure.